integration_tests/aave_abi.go

```diff
@@ -0,0 +1,1757 @@
+// Code generated - DO NOT EDIT.
```

Comment:
How was this generated? Also wondering if it actually gets referenced anywhere, since it seems like the reference to functions calls/topics are generated by hand in `cork_test`.

Reply:
Artifact from when I was trying to test for the event a different way. Will delete.